check_http_json.rb

```diff
@@ -301,6 +300,11 @@ def parse_args(options)
             options[:status_level] = x.split(',')
         end
 
+        options[:status_level_default] = 1
+        opts.on('--status_level_default INT', 'The default return code for not-expected http status codes. Defaults to 1.') do |x|
```

Comment:
```suggestion
        opts.on('--status_level_default VALUE', 'The default return code for unexpected HTTP status codes. Defaults to 1.') do |x|
```

You'll also need to add this to the `Usage` section of the README. 😄